src/many/src/message/error.rs

```diff
@@ -153,6 +154,12 @@ many_error! {
             => "The message's timestamp is out of the accepted range of the server.",
     -1006: RequiredFieldMissing as required_field_missing(field)
             => "Field is required but missing: '{field}'.",
+    -1007: SnapshotNotFound as snapshot_not_found(id)
```

Comment:
These are not generic errors, they're part of the 1000 attribute and should be defined there as attribute errors.

Reply:
Please address this. `-1000` is not the attribute ID. Take a look here to see how to define attribute specific errors: https://github.com/the-lifted-initiative/many-rs/blob/main/src/many/src/server/module/_9_account/errors.rs#L3-L9